build.sbt

```diff
@@ -128,6 +128,8 @@ lazy val drsFileSystem = (project in file("filesystems/drs"))
 
 lazy val databaseSql = (project in file("database/sql"))
   .withLibrarySettings("cromwell-database-sql", databaseSqlDependencies)
+  .dependsOn(core)
```

Comment:
Code smell 👃...  `core` is way too big/tempting of a large surface area of "stuff", usually associated with logic / data objects outside of SQL mappings. The `database` layer is supposed to stay as-thin-as-possible, only translating scala to slick/SQL, and leaving the logic to run in the service layer.

Comments continue below...